i18n/de.json

```diff
@@ -2,7 +2,7 @@
     "PrivateBin": "PrivateBin",
     "%s is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted %sin the browser%s using 256 bits AES.": "%s ist ein minimalistischer, quelloffener \"Pastebin\"-artiger Dienst, bei dem der Server keinerlei Kenntnis der Inhalte hat. Die Daten werden %sim Browser%s mit 256 Bit AES ver- und entschlüsselt.",
     "More information on the <a href=\"https://privatebin.info/\">project page</a>.": "Weitere Informationen sind auf der <a href=\"https://privatebin.info/\">Projektseite</a> zu finden.",
-    "Because ignorance is bliss": "Unwissenheit ist ein Segen",
+    "Because ignorance is bliss": "Was ich nicht weiß, macht mich nicht heiß",
```

Comment:
@rugk & @cryptomilk, I find the German translation of the "ignorance is bliss" slogan awkward and clunky. In other languages other phrases with similar meaning were chosen (see for example #540). Do you mind if we change this in German? Is my suggestion Ok or can you think of a better German phrase?

Reply:
I agree.